src/solve.jl

```diff
@@ -0,0 +1,16 @@
+@inline function (\){T}(A::SMatrix{2,2,T,4}, b::SVector{2,T})
+    d = det(A)
+    @inbounds x = SVector( A[2,2]*b[1] - A[1,2]*b[2],
+                           A[1,1]*b[2] - A[2,1]*b[1] ) / d
+    return x
+end
+
+@inline function (\){T}(A::SMatrix{3,3,T,9}, b::SVector{3,T})
+    d = det(A)
+    @inbounds begin
+        x = SVector( A[1,2]*A[2,3]*b[3] - A[1,2]*A[3,3]*b[2] - A[1,3]*A[2,2]*b[3] + A[1,3]*A[3,2]*b[2] + A[2,2]*A[3,3]*b[1] - A[2,3]*A[3,2]*b[1],
+                    -A[1,1]*A[2,3]*b[3] + A[1,1]*A[3,3]*b[2] + A[1,3]*A[2,1]*b[3] - A[1,3]*A[3,1]*b[2] - A[2,1]*A[3,3]*b[1] + A[2,3]*A[3,1]*b[1],
+                     A[1,1]*A[2,2]*b[3] - A[1,1]*A[3,2]*b[2] - A[1,2]*A[2,1]*b[3] + A[1,2]*A[3,1]*b[2] + A[2,1]*A[3,2]*b[1] - A[2,2]*A[3,1]*b[1] ) / d
```

Comment:
So this is basically inverse via cofactors, with the matvec multiply written out explicitly?  Seems reasonable to me, though did you compare speed-wise with `inv(A)*b`?